src/components/Navbar.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import styles from '../styles/Navbar.module.css';
+
+const Navbar = () => {
+  const links = [
+    {
+      path: '/',
+      element: 'Home',
+    },
+    {
+      path: 'calculator',
+      element: 'Calculator',
+    },
+    {
+      path: 'quotes',
+      element: 'Quotes',
+    },
+  ];
+  return (
+    <nav className={styles.navbar}>
+      <h2>Math Magicians</h2>
+      <ul className={styles.navlist}>
+        {links.map((link) => (
+          <li key={link.element}>
+            <NavLink className={styles.navlink} to={link.path}>
+              {link.element}
+            </NavLink>
```

Comment:
- [ ] I like how you are using `NavLink` instead of `Link` which makes it easier to apply active styles. Good job. :+1:

Reply:
Thank you :smile: